karma.conf.js

```diff
@@ -11,6 +11,14 @@ module.exports = function (config) {
     config.browsers.push('internet_explorer_11');
   }
 
+  config.files = [
+    // React
+    'https://scontent.xx.fbcdn.net/t39.3284-6/13591530_1796350410598576_924751100_n.js',
```

Comment:
If we used installed versions then perf tests aren't accurate because of dev mode (these are prod versions, so they perform better). What I'd like to do here is try minifying the local perf tests so that it does the same for incremental dom. I'll try and see what happens.